third_party/terraform/resources/resource_compute_region_instance_group_manager.go.erb

```diff
@@ -430,10 +430,10 @@ func waitForInstancesRefreshFunc(f getInstanceManagerFunc, d *schema.ResourceDat
 			log.Printf("[WARNING] Error in fetching manager while waiting for instances to come up: %s\n", err)
 			return nil, "error", err
 		}
-		if done := m.CurrentActions.None; done < m.TargetSize {
-			return done, "creating", nil
+		if m.Status.IsStable {
+			return nil, "created", nil
```

Comment:
Is changing this to `nil` safe? As I understand, https://github.com/hashicorp/terraform-plugin-sdk/blob/master/helper/resource/state.go#L126 implies that we'll fail to refresh if it's set to that.